config/openshift/patch.yaml

```diff
@@ -0,0 +1,24 @@
+# needed a patch because manifests.yaml gets overwritten by kustomize
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: validating-webhook-configuration
+  annotations:
+    # functionality only works on openshift
+    service.beta.openshift.io/inject-cabundle: "true"
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: webhook-service
+  namespace: system
+  annotations:
+    # functionality only works on openshift
+    service.alpha.openshift.io/serving-cert-secret-name: webhook-server-cert
```

Comment:
in other places we use the beta version (`service.beta.openshift.io/serving-cert-secret-name`), I guess we can upgrade that one too